specification/application_layer/functions.tex

```diff
@@ -107,25 +108,26 @@ \subsection{Network-wide time synchronization}
 \subsection{Primitive types and physical quantities}
 
 The namespaces \DSDLReference{uavcan.si} and \DSDLReference{uavcan.primitive}
-included in the standard data type set are designed to provide a very generic and flexible,
-albeit neither bandwidth-efficient nor low-latency, method of real-time data exchange.
+included in the standard data type set are designed to provide a generic and flexible
+method of real-time data exchange; however, these namespaces are not bandwidth-efficient
+or low-latency.
 
 Generally, applications where the bus bandwidth and latency are important should minimize their reliance
 on these generic data types and favor more specialized types instead that are custom-designed for their
 particular use cases; e.g., vendor-specific types or application-specific types, either
 designed in-house, published by third parties\footnote{As long as the license permits.}, or supplied by
 vendors of COTS equipment used in the application.
 
-Vendors of COTS equipment should always ensure that at least some minimal functionality is available
+Vendors of COTS equipment should always ensure that some minimal functionality is available
```

Comment:
This is one of those issues in the original document I've meant to fix in the final pass over the text later. The level of requirement is higher than it should be, can we please fix it here while we're at it?

```suggestion
Vendors of COTS equipment are recommended to ensure that some minimal functionality is available
```